extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -233,6 +233,15 @@ const BLOCK_SRA_COMBINERS_ = [
   },
   instances => getFirstInstanceValue_(instances,
       instance => instance.buildIdentityParams_()),
+  instances => {
+    let safeframeForced = false;
+    const forceSafeframes = [];
+    instances.forEach(instance => {
+      safeframeForced = safeframeForced || instance.forceSafeframe_;
+      forceSafeframes.push(Number(instance.forceSafeframe_));
+    });
+    return safeframeForced ? {'fsfs': forceSafeframes.join(',')} : null;
```

Comment:
What about dropping `safeframeForced `and just doing:

    return forceSafeframes.every((x) => x) ? {'fsfs': forceSafeframes.join(',')} : null;

Reply:
This would require reiterating the loop... having variable is slightly more performant.